lib/configuration.yaml

```diff
@@ -183,6 +183,24 @@ bench: &bench
 
 default: *dev
 
+##############################################################################
+##                                                                          ##
+##   Config for benchmarks of block validation                              ##
+##                                                                          ##
+##############################################################################
+
+bench-validation:
+  <<: *dev
+  core:
+    <<: *dev_core
+    genesis:
+      <<: *dev_core_genesis
+      spec:
+        <<: *dev_core_genesis_spec
+        protocolConstants:
+          <<: *dev_core_genesis_spec_protocolConstants
+          k: 5000
```

Comment:
What's the motivation for this?

Reply:
It's only used by the `cardano-sl-verification-bench-exe` executable which I used to profile.  It's reading the configuration file, and I needed a really high `k` value to be able to verify long lists of blocks.  This could be removed as one of the command options specifies which part of the configuration file to use, but I thought it might be worth having something that just runs, without modifying any config files.

Reply:
Alternatively, just take the `k` parameter itself as a command line argument?

Reply:
agreed